package.json

```diff
@@ -1,71 +1,77 @@
 {
-  "name": "graphql-fhir",
-  "version": "1.2.4",
-  "description": "A Javascript based GraphQL FHIR server",
-  "main": "index.js",
-  "author": "Robert-W <rwinterbottom@asymmetrik.com>",
-  "license": "MIT",
-  "private": false,
-  "scripts": {
-    "start": "cross-env NODE_ENV=production node src/index",
-    "nodemon": "cross-env NODE_ENV=development node src/scripts/nodemon",
-    "test": "cross-env NODE_ENV=test node src/scripts/test",
-    "prettier": "prettier \"src/**/*.js\" --write",
-    "lint": "eslint \"src/**/*.js\"",
-    "changelog": "conventional-changelog -p angular -s -i CHANGELOG.md"
-  },
-  "prettier": {
-    "trailingComma": "all",
-    "singleQuote": true
-  },
-  "dependencies": {
-    "@asymmetrik/fhir-gql-schema-utils": "^1.0.1",
-    "@asymmetrik/sof-graphql-invariant": "^1.0.2",
-    "@asymmetrik/sof-strategy": "^1.0.2",
-    "body-parser": "^1.18.3",
-    "compression": "^1.7.2",
-    "cross-env": "^5.2.0",
-    "express": "^4.16.3",
-    "express-graphql": "^0.6.12",
-    "glob": "^7.1.2",
-    "graphql": "^14.0.2",
-    "graphql-outfields": "^0.9.1",
-    "helmet": "^3.12.1",
-    "moment-timezone": "^0.5.21",
-    "nodemon": "^1.17.5",
-    "passport": "^0.4.0",
-    "passport-http-bearer": "^1.0.1",
-    "sanitize-html": "^1.18.2",
-    "superagent": "^3.8.3",
-    "validator": "^10.4.0",
-    "winston": "^3.0.0",
-    "xss": "^1.0.3"
-  },
-  "devDependencies": {
-    "conventional-changelog-cli": "^2.0.11",
-    "coveralls": "^3.0.2",
-    "eslint": "^5.1.0",
-    "jest": "^24.9.0",
-    "prettier": "^1.15.3"
-  },
-  "jest": {
-    "verbose": true,
-    "collectCoverage": true,
-    "coverageReporters": [
-      "text",
-      "lcov",
-      "json"
-    ],
-    "setupFiles": [
-      "<rootDir>/src/environment.js"
-    ],
-    "testEnvironment": "node",
-    "testPathIgnorePatterns": [
-      "<rootDir>/src/scripts/"
-    ],
-    "coveragePathIgnorePatterns": [
-      "<rootDir>/src/environment.js",
-      "<rootDir>/src/resources/"
-    ]
-  }
+	"name": "graphql-fhir",
+	"version": "1.3.1",
+	"description": "A Javascript based GraphQL FHIR server",
+	"main": "index.js",
+	"author": "Robert-W <rwinterbottom@asymmetrik.com>",
+	"contributors": [
+		"Eugene Koon <ekoon@asymmetrik.com>",
+		"Luan Tran <luan@asymmetrik.com>"
+	],
+	"license": "MIT",
+	"private": false,
+	"scripts": {
+		"start": "cross-env NODE_ENV=production node src/index",
+		"nodemon": "cross-env NODE_ENV=development node src/scripts/nodemon",
+		"test": "cross-env NODE_ENV=test node src/scripts/test",
+		"prettier": "prettier \"src/**/*.js\" --write",
+		"lint": "eslint \"src/**/*.js\"",
+		"changelog": "conventional-changelog -p angular -s -i CHANGELOG.md"
+	},
+	"prettier": {
+		"trailingComma": "all",
+		"singleQuote": true
+	},
+	"dependencies": {
+		"@asymmetrik/sof-graphql-invariant": "^2.0.0",
+		"@asymmetrik/sof-strategy": "^1.0.2",
```

Comment:
https://github.com/Asymmetrik/node-fhir-server-core/pull/305/files
I think the other PR says sof-strategy as 1.1.5